src/modules/MinifyImage/Module.js

```diff
@@ -0,0 +1,102 @@
+module.exports = function MinifyImage(options, UI) {
+  var output;
+  if (!options.inBrowser) {
+    base64Img = require('base64-img');
+    imagemin = require('imagemin');
+    imageminJpegtran = require('imagemin-jpegtran');
+    imageminPngquant = require('imagemin-pngquant');
+  }
+
+  function draw(input, callback, progressObj) {
+    progressObj.stop(true);
+    progressObj.overrideFlag = true;
+
+    var step = this;
+
+    function dataURItoBlob(dataURI) {
+      // convert base64 to raw binary data held in a string
+      // doesn't handle URLEncoded DataURIs - see SO answer #6850276 for code that does this
+      var byteString = atob(dataURI.split(',')[1]);
+
+      // separate out the mime component
+      var mimeString = dataURI.split(',')[0].split(':')[1].split(';')[0];
+
+      // write the bytes of the string to an ArrayBuffer
+      var ab = new ArrayBuffer(byteString.length);
+
+      // create a view into the buffer
+      var ia = new Uint8Array(ab);
+
+      // set the bytes of the buffer to the correct values
+      for (var i = 0; i < byteString.length; i++) {
+        ia[i] = byteString.charCodeAt(i);
+      }
+
+      // write the ArrayBuffer to a blob, and you're done
+      var blob = new Blob([ab], {
```

Comment:
This could be written just like `return new Blob([ia], { type: mimeString });` if you want.

Reply:
Thanks @IgorWilbert